### DISCUSSION TITLE
Templater Plugin Syntax Not Working

### DISCUSSION THREAD

**Discussion by margarita-rivera** [BryanJenksCommunity/FAQ#146] [Q&A]
Hey Bryan! Learning so much from your videos — though there's still learning curve for me, I am new to Obsidian and do not know any coding. 

### **Issue:** 
Templater syntax not working

### **Example:**

<%tp.date.now()%> remains <%tp.date.now()%> instead of:

<%tp.date.now()%> to 2022-04-23

I've uninstalled/installedyour templates and in loading a template (for example "YouTube") it still shows this way:

<img width="1244" alt="Screen Shot 2022-04-23 at 7 28 31 PM" src="https://user-images.githubusercontent.com/104243828/164892719-731ebeaa-44a9-46b3-a5e6-f61bf3a9c947.png">

### **Settings/Other actions taken:**
- Installed and enabled templater plugin, made sure it is updated (prompted by discussion #78 )
- Set templater folder to “Templates” folder and my settings as:
<img width="997" alt="Screen Shot 2022-04-23 at 7 29 40 PM" src="https://user-images.githubusercontent.com/104243828/164892743-6af96c86-2e98-4bba-808a-04c8c72f734a.png">

<img width="986" alt="Screen Shot 2022-04-23 at 7 29 48 PM" src="https://user-images.githubusercontent.com/104243828/164892746-8725f499-240b-456f-aca4-30bacbb35a7a.png">

- Turned off core plugin “templates” (prompted by discussion #60 ) 
- customized hotkeys of templater (prompted by discussion  #60)

<img width="996" alt="Screen Shot 2022-04-23 at 7 30 39 PM" src="https://user-images.githubusercontent.com/104243828/164892727-fc39b0be-9830-415c-b33c-bf89ff95ec4d.png">

- Made sure I uploaded latest vault.zip (prompted by the discussion in #85 ) and upon checking it's dated 2022-02-25

Would you have any idea what I'm doing wrong here? Been stuck on this issue for days! I’ve combed through forums, youtube tutorials and github and I still can’t seem to find any solution. But I'm not giving up in resolving this. 😅

Thanks in advance and thank you again for the resources and knowledge you share. Your channel is turning to become my go-to for any PKM and Obsidian-related content!

**Comment by tallguyjenks** [comment]
is the built in template plugin deactivated?

your screenshots and examples are very informative so thank you for making the question easier to respond to

**Reply by margarita-rivera** [reply]
Thanks for the response, yes I've deactivated the core template plugin thinking it will resolve this problem but unfortunately still doesn't seem to work 
<img width="1622" alt="Screen Shot 2022-04-24 at 11 50 41 AM" src="https://user-images.githubusercontent.com/104243828/164955806-6c8f80c4-f0da-40f9-a6ea-b24c7a66d63e.png">

**Reply by tallguyjenks** [reply]
you appear to be doing everything correctly, i'd suggest reaching out to the templater plugin developer via their git repo and see if its a technical issue, because it doesn't appear to be a workflow issue 🤔

**Reply by margarita-rivera** [reply]
Thanks so much for taking the time to read and your suggestion! Will do just that
